docs/reference/configuration/service-mesh/ingress.md

```diff
@@ -33,7 +33,7 @@ metadata:
 packages:
   - name: core
     repository: oci://ghcr.io/defenseunicorns/packages/uds/core
-    ref: 0.23.0-upstream
+    ref: 0.54.1-upstream
```

Comment:
I think in some places we have used `x.x.x` as a version placeholder instead of an explicit version (maybe with a comment of "check available versions <here>". Okay with just bumping all for now as well.